setup.php

```diff
@@ -33,7 +33,7 @@
 define('PLUGIN_TAG_VERSION', '2.12.1');
 
 // Minimal GLPI version, inclusive
-define("PLUGIN_TAG_MIN_GLPI", "10.0.0");
+define("PLUGIN_TAG_MIN_GLPI", "10.0.19");
```

Comment:
To avoid, as it blocks the plugin updates and consequently other fixes until the user is on the correct GLPI version, which, in this case, does not exist yet.
```suggestion
define("PLUGIN_TAG_MIN_GLPI", "10.0.0");
```